pennylane/ops/cv.py

```diff
@@ -796,33 +807,257 @@ class CatState(CVOperation):
     """
     num_wires = 1
     num_params = 3
-    par_domain = 'R'
-    grad_method = 'F'
-
-
-all_ops = [
-    Beamsplitter,
-    ControlledAddition,
-    ControlledPhase,
-    Displacement,
-    Kerr,
-    CrossKerr,
-    QuadraticPhase,
-    Rotation,
-    Squeezing,
-    TwoModeSqueezing,
-    CubicPhase,
-    Interferometer,
-    CatState,
-    CoherentState,
-    FockDensityMatrix,
-    DisplacedSqueezedState,
-    FockState,
-    FockStateVector,
-    SqueezedState,
-    ThermalState,
-    GaussianState
-]
+    par_domain = "R"
+    grad_method = "F"
+
+
+# =============================================================================
+# Observables
+# =============================================================================
+
+
+class MeanPhoton(CVObservable):
+    r"""pennylane.expval.MeanPhoton(wires)
+    Returns the photon number expectation value.
+
+    This expectation command returns the value
```

Comment:
Docstring will have to be updated to reflect refactor

Reply:
Fixed